setup.py

```diff
@@ -35,8 +35,7 @@ def get_version(rel_path):
         'profiling': [
             'pandas',
             'seaborn',
-            'matplotlib'
-        ]
+            'matplotlib']
```

Comment:
How is this cleaner code? This is the last nitpick :-)

Reply:
Sorry I added a dependency & then remove it. Didn't realize it changed the code.